symplyphysics/laws/thermodynamics/euler_relations/enthalpy_formula.py

```diff
@@ -0,0 +1,82 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    dimensionless,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    symbols
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.laws.thermodynamics import enthalpy_is_internal_energy_plus_pressure_energy as enthalpy_def
+from symplyphysics.laws.thermodynamics.euler_relations import internal_energy_formula
+
+# Description
+## The enthalpy differential can be integrated using the Euler's theorem on homogeneous functions for
+## internal energy.
+
+# Law: H = T * S + mu * N
+## H - enthalpy
+## T - temperature
+## S - entropy
+## mu - chemical potential
+## N - particle count
+
+# Note
+## - This formula works for single-component system. For many-component systems replace the RHS with
```

Comment:
Single-component system sounds weird. I suggest `system made of same type components`.

Reply:
![image](https://github.com/blackyblack/symplyphysics/assets/115647936/c43272df-6beb-42d1-9a74-244e68dc4678)
That's the technical name for it. `component` refers to the type of particles.